### PR TITLE
#2771: setup_clone.rst, added setup_clone.rst, updated README.md

### LINKED ISSUE #2771: #2771: Create 'setup_clone.rst' and link it in README.md
A `setup_clone.rst` file has to be created under 'documentation/configuration folder',
and respective documentation has to be fed in,
A link to `setup_clone.rst` file has to be placed in the main `README.md`.

### PR DESCRIPTION
**Pull request by rahul-kumi**
PR raised to resolve #2771 .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,75 +126,14 @@ shell access.
 Fork this project in your GitHub account.  Then, clone your repository, with
  one of the following approaches:
 
-- [simple clone](https://github.com/jeff1evesque/machine-learning/blob/master/README.md#simple-clone):
+- [simple clone](https://github.com/jeff1evesque/machine-learning/blob/master/documentation/configuration/setup_clone.rst#ssimple-clone):
```
Comment (jeff1evesque) [inline README.md]: `#ssimple-clone` typo should be changed to `#simple-clone`.

**Comment by jeff1evesque** [comment]
ff4b204: if you review your committed [`setup_clone.rst`](https://github.com/rahul-kumi/machine-learning/blob/ff4b204524a22bb6edfa8db0482cbd9c43c65bf3/documentation/configuration/setup_clone.rst), you'll notice we still haven't converted the `.md`, to `.rst` markdown syntax.

File: README.md (hunk 1 above)
Comment (rahul-kumi) [inline README.md]: i will clear this.

**Comment by rahul-kumi** [comment]
I will retain the document's structure and change the syntax to suit the `.rst` format.
would that do @jeff1evesque ?

File: README.md [hunk 2]
```diff
@@ -126,75 +126,14 @@ shell access.
...
-
-**Note:** `[release-tag]` corresponds to the [release tag](https://github.com/jeff1evesque/machine-learning/tags)
- value, used to distinguish between releases.
-
-**Note:** `[destination-directory]` corresponds to the desired directory path,
- where the project repository resides.  `[account]` corresponds to the git
- username, where the repository is being cloned from.  If the original
- repository was forked, then use your git username, otherwise, use
- `jeff1evesque`.
+ **Note**: various approaches of cloning can be found [here](https://github.com/jeff1evesque/machine-learning/blob/master/documentation/configuration/setup_clone.rst).
```
Comment (jeff1evesque) [inline README.md]: I would a minor language change, simply from the world of web development, it's generally frowned upon, when using words like *here* to signify something, somewhere else:

>
**Note**: various approaches of cloning can be found [here](https://github.com/jeff1evesque/machine-learning/blob/master/documentation/configuration/setup_clone.rst).

To the following:

>
**Note:** various [approaches](https://github.com/jeff1evesque/machine-learning/blob/master/documentation/configuration/setup_clone.rst) can be used to clone the repository, which would cover the possibility of cloning the repository, at a specific commit hash.

Comment (rahul-kumi) [inline README.md]: Done, @jeff1evesque .

File: README.md [hunk 3]
```diff
@@ -126,75 +126,14 @@ shell access.
...
-
-**Note:** `[release-tag]` corresponds to the [release tag](https://github.com/jeff1evesque/machine-learning/tags)
- value, used to distinguish between releases.
-
-**Note:** `[destination-directory]` corresponds to the desired directory path,
- where the project repository resides.  `[account]` corresponds to the git
- username, where the repository is being cloned from.  If the original
- repository was forked, then use your git username, otherwise, use
- `jeff1evesque`.
+ **Note**: various [approaches](https://github.com/jeff1evesque/machine-learning/blob/master/documentation/configuration/setup_clone.rst) can be used to clone the repository, which would cover the possibility of cloning the repository, at a specific commit hash.
```
Comment (jeff1evesque) [inline README.md]: There's some extra spacing, at the start of the line, and before ` **Note:**, which needs to be removed.

Comment (jeff1evesque) [inline README.md]: This is causing the line to be indented by a tab, when rendered.

Comment (rahul-kumi) [inline README.md]: I will check it.

**Pull request merged**